src/SDKs/EventGrid/DataPlane/Microsoft.Azure.EventGrid/Customization/Models/MediaJobOutputAsset.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.EventGrid.Models
+{
+    [JsonObject("#Microsoft.Media.JobOutputAsset")]
```

Comment:
@quintinb , I was wondering if this should have been auto-generated based on the Swagger (rather than getting added as a customization) - any ideas on why it didn't get auto-generated?

Reply:
Good catch.  Somehow the discriminator got removed from the MediaJobOutput when it got copied over.  I added it back and re-ran generated.cmd and the JsonObject marking was added as expected.  I originally thought this was a difference in using generate.cmd vs generate.ps1 but should have tested it.  I will submit a pull request to add the discriminator and once that is checked in update this.

Reply:
sounds good, thanks!